WordPress/Tests/WP/DeprecatedParametersUnitTest.inc

```diff
@@ -0,0 +1,64 @@
+<?php
+
```

Comment:
OK, so just a commented needed here then please.

(I was tempted to suggest putting these functions in alphabetical order, but then remembered seeing discussion about the better order - a comment would help tell others what the logic is.)